### DISCUSSION TITLE
tokyo-night theme

### DISCUSSION THREAD

**Discussion by azinsharaf** [monkeytypegame/monkeytype#3912] [Ideas] [answered]
can we have tokyo-night theme?

**Comment by azinsharaf** [comment, marked as answer]
> can we have tokyo-night theme?

I imported it as a custom theme.

**Reply by kaem-e** [reply]
could you share the link to it perhaps?

**Reply by Nordaar** [reply]
I use this one:

[monkeytype with theme](https://monkeytype.com?customTheme=eyJjIjpbIiMxYTFiMjYiLCIjN2FhMmY3IiwiI2MwY2FmNSIsIiM1NjVmODkiLCIjNDE0ODY4IiwiI2E5YjFkNiIsIiNmNzc2OGUiLCIjZmY5ZTY0IiwiI2ZmN2E5MyIsIiNlMGFmNjgiXX0=)

```
#1a1b26 background
#7aa2f7 main (accent)
#c0caf5 caret
#565f89 sub
#414868 sub alt
#a9b1d6 text
#f7768e error
#ff9e64 extra error
#ff9e64 colorful error
#e0af68 colorful extra error
```
